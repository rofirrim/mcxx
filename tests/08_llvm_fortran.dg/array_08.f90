! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_08.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    INTEGER, ALLOCATABLE :: A(:), B(:), C(:)
    INTEGER :: N, I

    N = 4
    ALLOCATE(A(N))
    ALLOCATE(B(N))
    ALLOCATE(C(N))

    DO I = 1, N
      A(I) = 1 + I
      B(I) = 2 + I
      C(I) = -I
    END DO

    C = A + B
    PRINT *, A
    PRINT *, B
    PRINT *, C
END PROGRAM MAIN
