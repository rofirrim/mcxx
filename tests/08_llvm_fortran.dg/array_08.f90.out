           2           3           4           5
           3           4           5           6
           5           7           9          11
